lib/nerves_hub/metrics.ex

```diff
@@ -153,6 +154,49 @@ defmodule NervesHub.EctoReporter do
   end
 end
 
+defmodule NervesHub.DeviceReporter do
+  require Logger
+
+  def events() do
+    [
+      [:nerves_hub, :devices, :connect],
+      [:nerves_hub, :devices, :disconnect],
+      [:nerves_hub, :devices, :update, :automatic]
+    ]
+  end
+
+  def handle_event([:nerves_hub, :devices, :connect], _, metadata, _) do
+    Logger.info("Device connected",
+      event: "nerves_hub.devices.connect",
+      device_identifier: metadata[:identifier],
```

Comment:
```suggestion
      identifier: metadata[:identifier],
```

I think since it's scoped to `devices` in the event key, we can match the metadata key here and save a few bytes. What do you think?

Reply:
Works for me!